README.md

```diff
@@ -1,4 +1,40 @@
 # adapt-navigationLogo
 
-AT Compatible
-Accessible
+Displays an image in the course navigation bar. Image can be configured to utilise three different sizings (small, large, and fill), whether the image should display on small screens, and if the image should be shown on a small screen whether an alternative, more mobile friendly, version should be used.
```

Comment:
Description needs updating – it still mentions sizings.

Reply:
All amends are done. If you're happy with the last two commits @tomgreenfield I'll merge the PR.